src/nft/components/collection/UnavailableCollectionPage.tsx

```diff
@@ -0,0 +1,67 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import { darken } from 'polished'
+import { AlertTriangle } from 'react-feather'
+import { Link } from 'react-router-dom'
+import styled, { css, useTheme } from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const Container = styled(Column)`
+  height: 75vh;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 48px;
+  gap: 8px;
+`
+const LinkStyles = css`
```

Comment:
do you not have these shared anywhere?

Reply:
Considered it, but in this case, it's relatively straightforward to manually query and test the component, which I've found to be more robust over time.

Reply:
Checking. If not, will add them to a shared location.

Reply:
nbd if not, can do in another pr